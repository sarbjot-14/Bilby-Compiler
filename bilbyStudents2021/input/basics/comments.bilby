main{
	% this is a comment%
	%%
	print 1.23 $n;  % testing comment%
	% this is a comment%
	
	% this comment ends in new line
	
	print 1.2222 $n;
	%%%%%
	print 1.3333 $n;
}