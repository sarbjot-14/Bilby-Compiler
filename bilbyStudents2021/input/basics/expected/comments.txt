1.230000
1.222200
1.333300
